search-page.lua

```diff
@@ -190,8 +194,8 @@ local function create_page(type, t)
         {"Ctrl+LEFT", "page_left_search", function() temp:move_page(-1, true) end, {}},
         {"Ctrl+RIGHT", "page_right_search", function() temp:move_page(1, true) end, {}},
         {"Ctrl+ENTER", "run_latest", function() temp:run_search(LATEST_SEARCH.keyword, LATEST_SEARCH.flags) end, {}},
-        {"f12", "open_search", function() temp:get_input(false) end, {}},
-        {"Shift+f12", "open_flag_search", function() temp:get_input(true) end, {}}
+        {o.keybind_open_search, "open_search", function() temp:get_input(false) end, {}},
+        {o.keybind_open_flag_search, "open_flag_search", function() temp:get_input(true) end, {}}
```

Comment:
Most of this PR is actually unnecessary. If we remove these two lines:

```suggestion
```

And change the `open_wrapper` function to be:

```lua
function list_meta:open_wrapper(advanced)
    if self.keyword == nil then self.empty_text = "Press f12 to enter search query" end
    local hidden = self.hidden
    if hidden then self:open() end
    if self.keyword == nil or not hidden then self:get_input(advanced) end
end
```

Then the standard mpv keybind to open the page will be also be used when the page is open instead of being overridden. Then the keybinds can be changed by using standard `input.conf` changes.